### PR TITLE
[breaking][libra framework] Add new events for dual attestion information changes

### LINKED ISSUE #5351: [Dual Attestation] Emit an event when compliance keys and base url are rotated

### PR DESCRIPTION
**Pull request by tzakian**
This PR adds additional events for rotation of compliance public key, and base url used for dual attestation checking.

It is breaking since it changes the`DualAttestation` resource.

closes #5351

### CODE REVIEW

File: json-rpc/docs/type_event.md [hunk 1]
```diff
@@ -72,17 +72,23 @@ Event data is serialized into one JSON object with a "type" field to indicate it
 | receiver            | string                   | Hex-encoded address of the account whose balance was credited by this deposit. |
 | metadata            | string                   | An optional field that can contain extra metadata for the event (from RawTransaction metadata??). This information can be used by an off-chain API to implement a sub-addressing scheme for a wallet. |
 
-#### sentpayment
```
Comment (xli) [inline json-rpc/docs/type_event.md:75]: deleted?

File: json-rpc/docs/type_event.md [hunk 2]
```diff
@@ -72,17 +72,23 @@ Event data is serialized into one JSON object with a "type" field to indicate it
...
+#### compliancekeyrotation
```
Comment (xli) [inline json-rpc/docs/type_event.md:75]: where we can find back these 2 new events key? In parentVasp account?

Comment (tzakian) [reply json-rpc/docs/type_event.md:75]: These two new event keys can be found in the dual attestation credential, I actually think I need to add these keys to that view. I'll update.

File: json-rpc/docs/type_event.md (hunk 1 above)
Comment (tzakian) [reply json-rpc/docs/type_event.md:75]: Good catch. I'll fix.

File: json-rpc/docs/type_event.md [hunk 3]
```diff
@@ -84,6 +84,26 @@ Event emitted when an account sends a payment.
 | receiver            | string                   | Hex-encoded address of the account whose balance was credited by this deposit. |
 | metadata            | string                   | An optional field that can contain extra metadata for the event (from RawTransaction metadata??). This information can be used by an off-chain API to implement a sub-addressing scheme for a wallet. |
 
+#### compliancekeyrotation
+
+Event emitted when the public key used for dual attestation checking on-chain is rotated.
```
Comment (xli) [inline json-rpc/docs/type_event.md:89]: Could you update there to mention where to find event key, and looks like only designated_dealer account can have it.

File: json-rpc/docs/type_event.md [hunk 4]
```diff
@@ -84,6 +84,26 @@ Event emitted when an account sends a payment.
...
+
+| Name                        | Type   | Description                                             |
+|-----------------------------|--------|---------------------------------------------------------|
+| type                        | string | constant string "compliancekeyrotation"                 |
+| new_compliance_public_key   | string | Hex-encoded new dual attestation compliance public key  |
+| time_rotated_seconds        | u64    | Blockchain time (in seconds) when the rotation occurred |
+
+#### baseurlrotation
+
+Event emitted when the url used for off-chain dual attestation checking is rotated on-chain.
```
Comment (xli) [inline json-rpc/docs/type_event.md:99]: same here, mention where to find the event key.

**Comment by xli** [comment]
approved for changes to json-rpc part

**Pull request closed without merging**